### PR TITLE
include Elixir Discord as part of "community"

### PR DESCRIPTION
**Pull request by Tuxified**
The Elixir Discord has grown to a considerable size (and Discord uses Elixir *wink wink*),
so I (think) it's due time to include it as a part of "the community"

### CODE REVIEW

File: guides/introduction/community.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ There are a number of places to connect with community members at all experience
   * We're on Freenode IRC in the [\#elixir-lang](http://webchat.freenode.net/?channels=elixir-lang) channel.
 
   * [Request an invitation](https://elixir-slackin.herokuapp.com/) and join the #phoenix channel on [Slack](https://elixir-lang.slack.com).
+  
+  * Feel free to join and checkout the #phoenix channel on [Discord](https://discord.gg/elixir).
```
Comment (Vechro) [inline guides/introduction/community.md:11]: Someone correct me if I'm wrong, but I think it should be "check out" (as two words) if you intend to use it as a verb.

Comment (scohen) [reply guides/introduction/community.md:11]: You are correct.

Comment (scohen) [inline guides/introduction/community.md:11]:
```suggestion
  * Feel free to join and check out the #phoenix channel on [Discord](https://discord.gg/elixir).
```

Comment (Tuxified) [reply guides/introduction/community.md]: Good catch, thanks

**Review by josevalim — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**